packages/svg/src/SvgFeatureRenderer/components/Subfeatures.js

```diff
@@ -0,0 +1,78 @@
+import { readConfObject } from '@gmod/jbrowse-core/configuration'
+import { observer } from 'mobx-react'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { chooseGlyphComponent, layOut, layOutFeature } from './util'
+
+function Subfeatures(props) {
+  const { feature, featureLayout, selected } = props
+
+  return (
+    <>
+      {feature.get('subfeatures').map(subfeature => {
+        const subfeatureId = String(subfeature.id())
+        const subfeatureLayout = featureLayout.getSubRecord(subfeatureId)
+        const { GlyphComponent } = subfeatureLayout.data
+        return (
+          <GlyphComponent
+            key={`glyph-${subfeatureId}`}
+            {...props}
+            feature={subfeature}
+            featureLayout={subfeatureLayout}
+            selected={selected}
+          />
+        )
+      })}
+    </>
+  )
+}
+
+Subfeatures.propTypes = {
+  feature: PropTypes.shape({ get: PropTypes.func.isRequired }).isRequired,
+  featureLayout: PropTypes.shape({
+    top: PropTypes.number.isRequired,
+    left: PropTypes.number.isRequired,
+    width: PropTypes.number.isRequired,
+    height: PropTypes.number.isRequired,
+  }).isRequired,
+  selected: PropTypes.bool,
+}
+
+Subfeatures.defaultProps = {
+  selected: false,
+  horizontallyFlipped: false,
+}
+
+Subfeatures.layOut = ({
+  layout,
+  feature,
+  bpPerPx,
+  horizontallyFlipped,
+  config,
+}) => {
```

Comment:
A function on the function?  Is this a thing?

Reply:
With the original `Box` glyph, `layOut` was a static method on the class component. When I refactored to hooks, this seemed to be the equivalent way to do it (similarly to how propTypes can be a static property of a class component or defined on the function).